action.yml

```diff
@@ -369,7 +369,7 @@ runs:
         )
 
         print $"\n(ansi purple)Installing workflow dependencies(ansi reset)"
-        mut uv_args = [sync --project $action_path --group action]
+        mut uv_args = [sync --project $action_path --group action --no-dev]
```

Comment:
I also discovered `uv` was installing `mypy` and `ruff` from the `dev` group. The `--no-dev` arg prevents `uv` installing dev group dependencies.